src/lib/components/support-card/support-card.svelte

```diff
@@ -88,35 +134,28 @@
       <div class="circle">
         <Heart style="height: 3rem; width: 3rem; fill: var(--color-primary)" />
       </div>
-      <div>
-        <ProjectAvatar {project} size="large" outline />
-      </div>
+      {#if project}
+        <div>
+          <ProjectAvatar {project} size="large" outline />
+        </div>
+      {/if}
     </div>
   </div>
   <h2 class="pixelated">Become a supporter</h2>
   <p>
-    {#if isSupportingProject}
-      You're already supporting <span class="typo-text-bold">{project.source.repoName}</span> with your
-      Drip List.
+    {#if isSupporting}
+      You're already supporting this with your Drip List.
     {:else}
-      Add <span class="typo-text-bold">{project.source.repoName}</span> to your Drip List to support
-      this project.
+      Add this {project ? 'project' : ''} to your Drip List to flexibly support it with an ongoing contribution.
```

Comment:
```suggestion
      Add this {project ? 'project' : ''} to {ownDripList === null ? 'a' : 'your'} Drip List where they will receive a portion of any funds streamed to your list.
```

Reply:
I think it sounds weird to use "where" here

Reply:
"**and** they will.."
would also do

Reply:
Yeah that's better. But maybe "and _it_ will" since it's referring to the list.

Reply:
It's referring to the project though (which receives a portion of what goes to the list). 'Become a supporter of the project by adding it to a Drip List'

Reply:
Ah, right this is used on project and drip list pages* So ya maybe a conditional for they/it

Reply:
It works for project and list. Just because it's a project doesn't mean it's multiple people.